src/pages/EditMonitor.vue

```diff
@@ -764,6 +764,30 @@
                                     </div>
                                 </div>
 
+                                <div class="my-3">
+                                    <div class="form-check">
+                                        <input id="saveResponse" v-model="monitor.saveResponse" class="form-check-input" type="checkbox">
+                                        <label class="form-check-label" for="saveResponse">
+                                            {{ $t("saveResponseForNotifications") }}
+                                        </label>
+                                    </div>
+                                    <div class="form-text">
+                                        <i18n-t key="saveResponseDescription" tag="div" class="form-text">
```

Comment:
currently does not work

```
runtime-core.esm-bundler.js:51 [Vue warn]: Unhandled error during execution of render function 
  at <I18nT key="saveResponseDescription" tag="div" class="form-text" > 
  at <BaseTransition appear=true persisted=false mode=undefined  ... > 
  at <Transition name="slide-fade" appear="" calculatedHeight=853 > 
  at <EditMonitor calculatedHeight=853 onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy(Object) > > 
  at <RouterView key="/add" calculatedHeight=853 > 
  at <Dashboard onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy(Object) > > 
  at <RouterView key=0 > 
  at <Layout onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy(Object) > > 
  at <RouterView > 
  at <App> 
  at <App>
```


```suggestion
                                        <i18n-t keypath="saveResponseDescription" tag="div" class="form-text">
```